plugins/usbdmx/ShowJockeyFactory.cpp

```diff
@@ -32,6 +32,9 @@ namespace usbdmx {
 
 using ola::usb::LibUsbAdaptor;
 
+const char ShowJockeyFactory::EXPECTED_MANUFACTURER[] = "Showjockey Co.,Ltd";
+const char ShowJockeyFactory::EXPECTED_PRODUCT[] =
+                                            "Showjockey Co.,Ltd.USB TO DMX51";
```

Comment:
It really is the DMX51. That's the string I got when I call for the product. I don't know why, but I think you guess it right : they probably run out of space.